diff --git a/dist/index.js b/dist/index.js
index 8383eb02ae716166f01a1f40f84866e0d4e47365..c928f605d56f50ce9f4d2d4f4cfcf94aeeedb165 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -11,7 +11,8 @@ import deepmerge from "deepmerge";
 import { findWorkspaceDir } from "@pnpm/find-workspace-dir";
 import { findWorkspacePackages } from "@pnpm/workspace.find-packages";
 import { packlist } from "@pnpm/fs.packlist";
-import { PackageCache } from "@embroider/shared-internals";
+import embroiderSharedInternals from "@embroider/shared-internals";
+const { PackageCache } = embroiderSharedInternals;
 tmp.setGracefulCleanup();
 var defaultFiles = {
   "index.js": `